js/src/personal/riyuan/components/home/Home.module.css

```diff
@@ -0,0 +1,5 @@
+.home{
```

Comment:
the tsx uses this as a id but this is a classname. BTW you can refer to IDs from css module too just like classname.